recipe/meta.yaml

```diff
@@ -2,6 +2,10 @@
 {% set sha256 = "ac46db6bfcaaec8cd28335231076815bd5438f401a4a05e33736b4f9ff12e59a" %}
 
 {% set mpi = mpi or 'mpich' %}
+{% set build = 1 %}
```

Comment:
@ltalirz Could you please move this line setting the build number right after the sha256 line above ?